etc/testing/arangod-common.conf

```
@@ -1,6 +1,6 @@
 [log]
 line-number = false
-force-direct = false
+force-direct = true
```

should not be part of this PR
```suggestion
force-direct = false
```